Example/LDOLayoutTemplates/ComplexViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ComplexViewController.swift
+//  LDOLayoutTemplates Example
+//
+//  Created by Sebastian Ludwig on 14.10.18.
+//  Copyright © 2018 Julian Raschke und Sebastian Ludwig GbR. All rights reserved.
+//
+
+import UIKit
+import LDOLayoutTemplates
+
+class ComplexViewController: UIViewController {
+    @IBOutlet weak var portraitLargeEventsLayout: LDOLayoutTemplate!
+    @IBOutlet weak var landscapeLargeEventsLayout: LDOLayoutTemplate!
+    private var defaultLayout: LDOLayoutTemplate!
+    private var largeListLayoutActive = false
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        defaultLayout = LDOLayoutTemplate(forCurrentStateBasedOn: portraitLargeEventsLayout)
+    }
+    
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        guard largeListLayoutActive else { return }
```

Comment:
+Comments for the `guard` and the transition below? Might help when users try to understand the source code without looking at the app at the very same time.